plugins/importexport/crossref/CrossRefExportPlugin.inc.php

```diff
@@ -120,19 +120,22 @@ function displayIssueList(&$templateMgr, &$journal) {
 		$this->registerDaoHook('IssueDAO');
 		$issueIterator =& $issueDao->getPublishedIssues($journal->getId(), Handler::getRangeInfo('issues'));
```

Comment:
Thanks @asmecher. I added and pushed it together with the first commit i.e. all again -- for users to be able to apply only one patch...